cmd/proxy.go

```diff
@@ -0,0 +1,94 @@
+package cmd
+
+import (
+	"github.com/MakeNowJust/heredoc"
+	"github.com/goto/shield/config"
+	"github.com/goto/shield/internal/proxy/envoy/xds"
+	"github.com/goto/shield/internal/store/postgres"
+	shieldlogger "github.com/goto/shield/pkg/logger"
+	"github.com/spf13/cobra"
+	cli "github.com/spf13/cobra"
+)
+
+func ProxyCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "proxy <command>",
+		Short: "Proxy management",
+		Long:  "Server management commands.",
+		Example: heredoc.Doc(`
+			$ shield proxy envoy-xds start -c ./config.yaml
```

Comment:
```suggestion
			$ shield proxy envoy start -c ./config.yaml
```
We could keep envoy for now
Later if we want to extend, we could have subcommand envoy xds start
But the default is xds if we use envoy start